src/main/Main.hs

```diff
@@ -605,9 +606,11 @@ buildCmd opts go = do
 
 uninstallCmd :: [String] -> GlobalOpts -> IO ()
 uninstallCmd _ go = withConfigAndLock go $ do
-    $logError "stack does not manage installations in global locations"
-    $logError "The only global mutation stack performs is executable copying"
-    $logError "For the default executable destination, please run 'stack path --local-bin'"
+    $prettyErrorL . (++ [cmd]) . concatMap wordDoc $
```

Comment:
How about

```haskell
    $prettyErrorL
      [ flow "stack does not manage installations in global locations."
      , flow "The only global mutation stack performs is executable copying."
      , flow "For the default executable destination, please run"
      , shellColor "stack path --local-bin"
      ]
```

No need to use `fromString` explicitly since OverloadedStrings is on.  I think it'd be good if most of the larger logging looked like this, because it's clear how to add some new `Doc` to the middle of it.  Whereas if `concatMap flow` is used, things need to be refactored for it to be modified.